database/migrations/2014_10_12_000001_create_password_reset_tokens_table.php

```diff
@@ -9,11 +9,11 @@
 return new class() extends Migration {
     public function up(): void
     {
-        if (Schema::hasTable('password_resets')) {
```

Comment:
Do we need a migration step for this? Or some sort of documentation for queries we need to run on our dumps?

Reply:
`php artisan migrate` should be the only required command to run in this case